crates/compiler/gen_dev/src/object_builder.rs

```diff
@@ -230,6 +353,15 @@ fn build_object<'a, B: Backend<'a>>(
                 "roc_shm_open".into(),
                 "shm_open".into(),
             );
+        } else if matches!(output.format(), BinaryFormat::Coff) {
+            // TODO figure out why this symbol is required, it should not be required
+            // Without this it does not build on Windows
+            generate_wrapper(
+                &mut backend,
+                &mut output,
+                "roc_getppid".into(),
+                "malloc".into(),
```

Comment:
wrapping malloc?

Reply:
yeah, `roc_getppid` was used somehow in the builtins (I think)? so we needed to provide the symbol. I'll dig into this a bit, because it should not be required

Reply:
I suspect it is these functions from `expect.zig`

```
extern fn roc_shm_open(name: *const i8, oflag: c_int, mode: c_uint) c_int;
extern fn roc_mmap(addr: ?*anyopaque, length: c_uint, prot: c_int, flags: c_int, fd: c_int, offset: c_uint) *anyopaque;
extern fn roc_getppid() c_int;
```